#VESTA.BAY# FULL PORT (#2)
VestaOfOrion/Vesta.Bay: README.md

> # baystation12

Reviewer: ```suggestion
# Vesta of Orion
```

> [Website](http://baystation12.net/) - [Code](http://github.com/Baystation12/Baystation12/) - [Discord](https://discord.baystation12.net/) - [IRC](https://kiwiirc.com/client/irc.sorcery.net/codershuttle): irc://irc.sorcery.net/#codershuttle

Reviewer: ```suggestion
This is Vesta of Orion's repository based on the [Baystation12](http://github.com/Baystation12/Baystation12/) codebase.

[Code](https://github.com/VestaOfOrion/Vesta.Bay) - [Organization](https://github.com/VestaOfOrion) - [Discord](https://discord.gg/sBzsswR)

```

### LICENSE

> The code for Baystation12 is licensed under the [GNU Affero General Public License v3](http://www.gnu.org/licenses/agpl.html), which can be found in full in [/LICENSE](/LICENSE).

Reviewer: ```suggestion
The code for Vesta of Orion is licensed under the [GNU Affero General Public License v3](http://www.gnu.org/licenses/agpl.html), which can be found in full in [/LICENSE](/LICENSE).

```
